DDD.Functions.Extensions/DDD.Functions.Extensions.csproj

```diff
@@ -1,11 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="microsoft.net.sdk.functions" Version="1.0.26" />
+    <PackageReference Include="microsoft.net.sdk.functions" Version="3.0.3" />
```

Comment:
It's worth reading the breaking change logs of azure functions between these versions to see if this is likely to break anything